Server/VikingVault.Services/UserService.cs

```diff
@@ -63,5 +64,24 @@ private BankAccount CreateBankAccount(User user, String currencyType)
                 Balance = 0.0f
             };
         }
+
+        public void DeleteUser(UserEmail userEmail)
+        {
+            try
+            {
+                User user = this.GetByEmail(userEmail.Email);
+                _dbContext.Remove(user);
+                _dbContext.SaveChanges();
+            }
+            catch (Exception e)
+            {
+                throw new UserServiceException();
+            }
+        }
+
+        public User GetByEmail(string email)
```

Comment:
If this method is not used anywhere else, it's a bit redundant to do it this way, for now it's better to use (if it's only used here) `User user = _dbContext.User.SingleOrDefault(user => user.Email == email);`